torch/_dynamo/variables/misc.py

```diff
@@ -815,8 +815,12 @@ def call_function(
         from .tensor import NumpyNdarrayVariable
 
         options = VariableTracker.propagate([[self]], [args], [list(kwargs.values())])
-        # lookup method name in torch_np
-        if hasattr(torch_np, self.value.__name__):
+        # lookup method name in torch_np. Things like np.dtype(float) are not supported yet.
+        if self.value.__name__ == "dtype":
+            unimplemented(
+                f"numpy dtype function is not supported yet. Got {self.value}."
+            )
+        elif hasattr(torch_np, self.value.__name__):
             func = getattr(torch_np, self.value.__name__)
             return wrap_fx_proxy_cls(
```

Comment:
The contract here is sus to me? any function on any numpy variable always returns a NumpyNdarrayVariable we pack with tensor types? This is totally out of scope for your PR, something I need to follow up on.

Reply:
Yeah you are right, it's not guaranteed that any function on any numpy variable always returns a NumpyNdarrayVariable. This is definitely still WIP.